frontend/src/components/SearchBar/index.jsx

```diff
@@ -1,24 +1,41 @@
 import React, { Component } from "react";
 import { Button } from "reactstrap";
+
+import ResourceCard from "../../components/ResourceCard";
 import "./styles.scss";
 
 class Search extends Component {
   constructor(props) {
     super(props);
-    this.state = { value: "" };
+    this.state = { value: "", showResults: false };
   }
 
+  renderCards = card => <ResourceCard title={card.title} text={card.text} />;
+
+  handleSubmit = e => {
```

Comment:
Not a change, but rather a heads up: to improve user experience, we'll likely want to (at some point) [_debounce_](https://levelup.gitconnected.com/debounce-in-javascript-improve-your-applications-performance-5b01855e086) the input, meaning that we'll incrementally run queries as the user types.